pages/index.jsx

```diff
@@ -1,10 +1,20 @@
 var React = require('react');
 
+var Habitat = require('habitat');
+Habitat.load();
+var env = new Habitat();
+
+var cookieDomain = env.get('FULL_SUBDOMAIN_FOR_COOKIE');
+
 var Index = React.createClass({
   render: function() {
     return (
       <html>
         <head>
+          <script>
+            window['optimizely'] = window['optimizely'] || [];
+            window['optimizely'].push(["setCookieDomain", {{cookieDomain}}]);
+          </script>
```

Comment:
This won't work with React, as it gets rendered like this:

```js
<script>window[&#x27;optimizely&#x27;] = window[&#x27;optimizely&#x27;] || []; window[&#x27;optimizely&#x27;].push([&quot;setCookieDomain&quot;, localhost]);</script>
```

What's the best way to add this in React?

@ScottDowne or @alicoding ?